homeassistant/components/eq3btsmart/config_flow.py

```diff
@@ -0,0 +1,94 @@
+"""Config flow for eQ-3 Bluetooth Smart thermostats."""
+
+import logging
+from typing import Any
+
+from homeassistant import config_entries
+from homeassistant.components.bluetooth import BluetoothServiceInfoBleak
+from homeassistant.const import CONF_MAC, CONF_NAME
+from homeassistant.data_entry_flow import FlowResult
+from homeassistant.helpers.device_registry import format_mac
+
+from .const import CONF_RSSI, DOMAIN
+from .schemas import SCHEMA_NAME, SCHEMA_NAME_MAC
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class EQ3ConfigFlow(config_entries.ConfigFlow, domain=DOMAIN):
+    """Config flow for eQ-3 Bluetooth Smart thermostats."""
+
+    VERSION = 1
+
+    def __init__(self) -> None:
+        """Initialize the config flow."""
+
+        self.name = ""
+        self.mac = ""
+        self.rssi = 0
+
+    async def async_step_user(
+        self, user_input: dict[str, Any] | None = None
+    ) -> FlowResult:
+        """Handle a flow initialized by the user."""
+
+        errors: dict[str, str] | None = {}
```

Comment:
```suggestion
```
not used, but is there a way to validate the user entered configuration is even valid?